src/P2P/Seed/Seeder.cpp

```diff
@@ -129,7 +129,22 @@ void Seeder::Accept(const asio::error_code& ec)
             SocketPtr pSocket(new Socket(SocketAddress::FromEndpoint(m_pSocket->remote_endpoint()), m_pAsioContext, m_pSocket));
             pSocket->SetOpen(true);
 
-            auto pPeer = m_peerManager.Write()->GetPeer(pSocket->GetIPAddress());
+            const IPAddress& ipAddress =  pSocket->GetIPAddress();
+
+            const std::string& ipAddressStr = ipAddress.GetAddress().to_string();
+
+            if(Global::GetConfig().IsPeerBlocked(ipAddressStr)) {
```

Comment:
nit: C++ devs nearly always include a space between `if` and `(`. I would recommend sticking with that formatting standard.